client/src/components/activities/ActivityOverview/useActivityOverviewFilter.ts

```diff
@@ -0,0 +1,42 @@
+import { offsetIfOverflowing } from "@/components/layout/Header/LogbookMenu/floating-middleware";
+import {
+	autoUpdate,
+	shift,
+	useClick,
+	useDismiss,
+	useFloating,
+	useInteractions,
+	useRole
+} from "@floating-ui/react";
+import { useState } from "react";
+
+export default function useActivityOverviewFilter() {
```

Comment:
I don't like this hook name. It's just the floating state, and the name of the hook should make that clear.